spec/record_net_http_spec.rb

```diff
@@ -127,7 +127,7 @@ def stop_recording
         :message:
         - :name: ary
           :class: Array
-          :value: '["1", "2"]'
+          :value: "[1, 2]"
```

Comment:
Those stringification changes don't look good to me. Before them, the values where valid Ruby literals. after the change they're not only invalid, but also ambiguous (consider `[1, 2]` -- `[1, 2]`? `["1", 2]`? `[1, "2"]`? or `{:foo: 4}` -- `{foo: '4'}`? `{":foo" => 4}`? ...).

Reply:
This should really not be language-specific though. The most likely place for the user to see this data is in the AppMap view. The hash-rocket really didn't make sense as a formatting. Maybe something closer to JSON - but overall the goal here is to make `stringify` fast, because it takes a lot of the time in AppMap.

Reply:
> This should really not be language-specific though.

Why not? The user presumably knows the language and the principle of least surprise dictates it to be in the usual language-specific form. Using the `{a: 3}` shorthand for symbol keys is fine. `{:a: 3}` is the worst of both worlds, plus the ambiguity can be confusing and potentially important.

> but overall the goal here is to make stringify fast, because it takes a lot of the time in AppMap.

Did you measure if it's an improvement? It's not entirely clear to me that it should be (maybe except limiting the number of elements).

Reply:
Also, it should simply limit the amount of elements displayed. With the changes, what's `[42]`? Real `[42]`? Or `['42']`? Or maybe `[42] * 42`? These all stringify to `[42]` after the changes.

Reply:
I've updated the stringification. Please take another look.